packages/rich-text-editor/docs/ToggleIconButton.stories.tsx

```diff
@@ -0,0 +1,41 @@
+import React from "react"
+import { withDesign } from "storybook-addon-designs"
+import boldIcon from "@kaizen/component-library/icons/bold.icon.svg"
+import { CATEGORIES, SUB_CATEGORIES } from "../../../storybook/constants"
+import { figmaEmbed } from "../../../storybook/helpers"
+import {
+  ToggleIconButton,
+  ToggleIconButtonProps,
+} from "../src/components/ToggleIconButton/ToggleIconButton"
+
+export default {
+  title: `${CATEGORIES.components}/${SUB_CATEGORIES.richTextEditor}/Toggle Icon Button`,
+  component: ToggleIconButton,
+  parameters: {
+    actions: {
+      argTypesRegex: "^on.*",
+    },
+    docs: {
+      description: {
+        component:
+          'import { ToggleIconButton } from "@kaizen/rich-text-editor"',
+      },
+    },
+    ...figmaEmbed(
+      "https://www.figma.com/file/eZKEE5kXbEMY3lx84oz8iN/%E2%9D%A4%EF%B8%8F-UI-Kit%3A-Heart?node-id=12317%3A92573"
+    ),
+  },
+  decorators: [withDesign],
+}
+
+export const DefaultStory = (args: ToggleIconButtonProps) => (
```

Comment:
Not sure if you've tested other icons, but might be worth including the relevant ones in a story to ensure they all render correctly (eg. alignment next to each other, whether the current styles work with all icons or if it looks off when changed).

Reply:
Might hold off on producing more stories for this individual component for now until we can the Toolbar and ToolbarSection (also design feedback on the button itself)